### ISSUE TITLE
The document doesn't mention how to create class member functions in rust.

### ISSUE THREAD

**Issue by shi-yan** [neon-bindings/neon#837]
The object second of the document is kinda sketchy. it doesn't mention how to create class member functions in rust?

also, can a closure be passed in as a function argument?

I found this example:

https://github.com/neon-bindings/examples/tree/main/examples/async-sqlite

but it doesn't work for me.

for example, I created the function databaseNew

but it will be recognized as a normal function:

```bash
Welcome to Node.js v16.13.1.
Type ".help" for more information.
> const db = require(".")
undefined
> db
{ databaseNew: [Function (anonymous)] }
> db()
Uncaught TypeError: db is not a function
```

not a member function

**Comment by kjvalencik** [comment]
You cannot create classes directly from Rust with Neon (there is an RFC proposal for this feature, but no implementation).

It requires some glue code in JavaScript. The idea is to create a wrapper class in JavaScript that delegates to the appropriate Neon functions.

Here is the glue code from the example:

https://github.com/neon-bindings/examples/blob/main/examples/async-sqlite/index.js

**Comment by kjvalencik** [comment]
Creating closures from functions is a relatively new feature and only available on the prerelease of `0.10`.

https://docs.rs/neon/0.10.0-alpha.3/neon/types/struct.JsFunction.html#method.new

**Comment by shi-yan** [comment]
Thank you very much!

I wish there could be an example on how to create member functions on the official document.
